HappyAnding/HappyAnding/Views/ShortcutsZipView.swift

```diff
@@ -16,10 +17,21 @@ struct ShortcutsZipView: View {
     @AppStorage("isReauthenticated") var isReauthenticated = false
     @AppStorage("useWithoutSignIn") var useWithoutSignIn: Bool = false
     
+    @State var isGradeAlertShowing = false
+    
     var body: some View {
         if signInStatus {
-            ShortcutTabView()
-                .environmentObject(userAuth)
+            ZStack {
+                ShortcutTabView()
+                    .environmentObject(userAuth)
+                
+                if isGradeAlertShowing {
+                    GradeAlertView(isShowing: $isGradeAlertShowing)
+                }
+            }
+            .onChange(of: gradeAlerter.isPresented) { newValue in
+                isGradeAlertShowing = newValue
+            }
```

Comment:
@JMM00 
withAnimation 내부에서 상태를 변경하면 사용하면 됩니다!
그런데 이렇게 사용하면 원래는 뷰가 사라질 때에도 자연스럽게 사라져야하는데, 왜 적용이 안 되는지는 저도 더 분석해봐야할 것 같아요

```suggestion
            .onChange(of: gradeAlerter.isPresented) { newValue in
                withAnimation {
                    isGradeAlertShowing = newValue
                }
            }
```

Reply:
저도 이상하게 onChange에서는 뷰가 사라질때 값 변경은 감지를 못해서 Binding변수로 따로 처리해줬습니다. 이것과 연관이 있을 수도 있겠군요ㅠㅠ